### PR TITLE
Bug 1788832: check ServiceCatalog basic usages in OCP 4.3

### PR DESCRIPTION
**Pull request by jianzhangbjz**
As title, do some basic tests, such as, enable/disable ServiceCatalog, deploy a fake broker, etc.

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
@shawn-hurley @jmrodri Could you help give the approval and merge it? Thanks!

File: test/extended/operators/service_catalog.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+				output, err := oc.AsAdmin().Run("get").Args("ns", v.namespace).Output()
+				if err != nil {
+					if strings.Contains(output, "not found") {
+						e2e.Logf("Disable %s successfully.", v.object)
+						return true, nil
+					}
+				}
+				return false, nil
+			})
+			o.Expect(err).NotTo(o.HaveOccurred())
```
Comment (shawn-hurley) [inline test/extended/operators/service_catalog.go]: Can we make sure that there are no leftover resources when this occurs such as the `APIService`?

Comment (jmrodri) [reply test/extended/operators/service_catalog.go]: Shawn is correct, after switching to `Removed` we should see if the `APIService` is still there or the namespaces for the service catalog items: `openshift-service-catalog-controller-manager` and `openshift-service-catalog-apiserver`. Though I would be okay for that to be a follow-on PR.

**Comment by jmrodri** [comment]
@jianzhangbjz I think this test should check to see if service catalog apiserver and controller-manager are firing the appropriate alerts. Then I think this PR would be good.

**Comment by jmrodri** [comment]
Will need the master PR to merge first https://github.com/openshift/origin/pull/24507

File: test/extended/operators/service_catalog.go (hunk 1 above)
Comment (jianzhangbjz) [reply test/extended/operators/service_catalog.go]: @shawn-hurley @jmrodri Thanks! Sorry for the late, add the `APIservice` checking. We have already covered the namespaces checking.

**Comment by jianzhangbjz** [comment]
@shawn-hurley @jmrodri Could you help have a review and add the /approval and /lgtm labels? Thanks!

**Comment by jianzhangbjz** [comment]
@shawn-hurley Could you help give approval? Thanks!

**Pull request closed without merging**